examples/compiled/time_output_utc_timeunit.vg.json

```diff
@@ -56,7 +57,7 @@
   "scales": [
     {
       "name": "x",
-      "type": "time",
+      "type": "utc",
       "domain": {
         "data": "data_0",
         "field": "utcyearmonthdatehoursminutes_date"
```

Comment:
Is it enough to append _date? Do we also need the time zone as part of the name in case a user has both utc and local time for a date in the same chart. This could be a follow-up fix.

Reply:
Never mind, we already add it.